### ISSUE TITLE
Daily meeting (16/12/2016)

### ISSUE THREAD

**Issue by Lai0723** [Lai0723/Gallery-Shopping-Website#5]
So anything to report today?

**Comment by kcchan3012** [comment]
Today i changing some design , i decide to put a slide show to let user to preview some of the gallery and I uploaded.

**Comment by boonsenglim** [comment]
What a great idea, by the way i just finish with the registration function just now and i had upload it . If got any problem please tell me. So ben you can start your part.

**Comment by pqyyieeuyi** [comment]
Ok, so i need validate the user login with the data in the userlogin database right?

**Comment by Lai0723** [comment]
Yup, you must validate with the database if valid then log in to the account if not match any record with the database prompt invalid password or username.

**Comment by Lai0723** [comment]
By the way , i will work on the changing password and maybe will upload tomorrow or sunday.

**Comment by pqyyieeuyi** [comment]
Ok, thank wei chun i will try it now. Maybe tomorrow I will upload it.

**Comment by Lai0723** [comment]
So i think nobody have any question. So today meeting until here, all can dismiss and work on your development.
